src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.service;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionService {
+    MovieSession add(MovieSession movieSession);
+
+    MovieSession get(Long id);
```

Comment:
According to the checklist, the `get()` method on the service layer should return the entity directly instead of `Optional`. You should adjust the return type to `MovieSession` and handle the `Optional` at the service layer, ensuring that `NoSuchElementException` is not thrown.

Reply:
+